### PR TITLE
Code of Conduct

### PR DESCRIPTION
**Pull request by gsaslis**
Adds code of conduct.
Ruby style.
It can be changed in the future if the team decides to do so 😄

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## 3Scale Community Code of Conduct v1.0
```
Comment (mikz) [inline CODE_OF_CONDUCT.md]: 3scale

**Comment by gsaslis** [comment]
it looks like an alternative code of conduct we can adapt is the [Ansible CoC](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html)

do you all think that's a more appropriate basis?

**Comment by Martouta** [comment]
I like it. Please @gsaslis add it and then let's ask in System channel what people think.

**Comment by gsaslis** [comment]
@Martouta let's reach consensus first  ; )

**Comment by thomasmaas** [comment]
I like the tone of it. Needs some edits but seems a good candidate to start from.

**Comment by secretsurf** [comment]
LGTM

**Pull request merged**